hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/MemStoreLABImpl.java

```diff
@@ -264,33 +270,31 @@ public void close() {
     }
     // We could put back the chunks to pool for reusing only when there is no
     // opening scanner which will read their data
-    int count  = openScannerCount.get();
-    if(count == 0) {
-      recycleChunks();
-    }
+    this.refCnt.release();
   }
 
-  int getOpenScannerCount() {
-    return this.openScannerCount.get();
+  /**
+   * This method is only used for test.
+   * @return
+   */
+  int getRefCntValue() {
```

Comment:
Use RestrictedApi. And also remove the empty `@return` block.

Reply:
@Apache9 , thank you very much for review , already used RestrictedApi to fix the problem.